spec/CampMinder/base64_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe CampMinder::Base64 do
+  describe ".urlsafe_encode64" do
+    it "encodes the payload in base64" do
+      expect(Base64).to receive(:urlsafe_encode64).and_return("LOLOL===")
+      expect(CampMinder::Base64.urlsafe_encode64("lol")).to eq "LOLOL"
+    end
+  end
+
+  describe '.urlsafe_decode64' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.